src/components/CreateNicRow/CreateNicRow.js

```diff
@@ -53,12 +53,12 @@ const getUsedNetworks = vm => {
   return usedNetworks;
 };
 
-const getNetworkChoices = (vm, networks) => {
+const getNetworkChoices = (vm, networks, namespace) => {
   const usedNetworks = getUsedNetworks(vm);
   const networkChoices = networks
     .filter(
       network =>
-        network.metadata.namespace === vm.metadata.namespace &&
+        (!namespace || network.metadata.namespace === namespace) &&
```

Comment:
please use `getNamespace` method here as well

Reply:
@suomiy , per @rawagner comment https://github.com/kubevirt/web-ui-components/pull/316#issuecomment-480741538 I'm going to remove this code, it will move to web-ui repo via https://github.com/kubevirt/web-ui/pull/265